### DISCUSSION TITLE
Why Extinction coefficient is a constant

### DISCUSSION THREAD

**Discussion by KeLiu7** [APSIMInitiative/ApsimX#9038] [Q&A]
Why `ExtinctionCoeff` is a constant (0.6) in the latest APSIM Next Generation version? Is there a reason to do so? I remember it should vary with the growth stage in the earlier versions.

**Comment by hut104** [comment]
Hi,
Which crop are you considering?

We used to have that capacity, but it was not often used (ie the same value was used for each growth stage).  There is flexibility in PMF to allow changes to canopy architecture such as this if we have the relevant data.

**Reply by KeLiu7** [reply]
> Hi, Which crop are you considering?
> 
> We used to have that capacity, but it was not often used (ie the same value was used for each growth stage). There is flexibility in PMF to allow changes to canopy architecture such as this if we have the relevant data.

Thanks Neil. It's Barley module.

Biomass accumulation is very sensitive to extinction coefficient. Perhaps we should keep the previous multiplication function, we do have some data sets, I will test it.
